projects/angular/storyteller/src/app/routes/tracking/tracking.component.ts

```diff
@@ -20,55 +27,66 @@ const MAX_PAUSE_INTERVAL: number = 5 * 60 * 1000; // 5 min
 })
 export class TrackingComponent implements OnDestroy {
   selectedStory: Story;
-  storySelect = new FormControl();
-  tracks: string[] = [];
+  trackRows: TrackRow[] = [];
   storyDurationTime: string;
   trackSub = new Subscription();
+  isTrackingLoading: boolean = false;
 
   constructor(
     public loadingService: LoadingService,
     public userService: UserService,
     private firebaseService: FirebaseService,
   ) {
     this.loadingService.isLoading = false;
-    this.storySelect.valueChanges.subscribe((story: Story) => {
-      this.trackSub = this.firebaseService.getTracks(story.getId()).subscribe(tracks => {
-        tracks.sort((a, b) => {
-          return Math.sign(a.getEndedMs() - b.getEndedMs());
-        });
+  }
+
+  storySelected(story: Story): void {
+    this.isTrackingLoading = true;
+    this.trackSub = this.firebaseService.getTracks(story.getId()).subscribe(tracks => {
+      tracks.sort((a, b) => {
+        return Math.sign(a.getEndedMs() - b.getEndedMs());
```

Comment:
Not sure you need `Math.sign()`. Isn't it positive/negative?

Reply:
```
Math.sign(*number < 0*) = -1
Math.sign(0) = 0
Math.sign(*number > 0*) = 1
```